src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -217,9 +217,11 @@ class CouchObjectProvider {
                 this.indicator.setIndicatorToState(DISCONNECTED);
                 console.error(error.message);
                 throw new Error(`CouchDB Error - No response"`);
-            }
+            } else {
+                console.error(error.message);
 
-            console.error(error.message);
+                throw error;
```

Comment:
Is it necessary to both `console.error` and `throw error`?

Reply:
The thinking here is that the specific details details of the Couch error should be logged, but the error should also bubble up to the API (and UI potentially).

Reply:
sounds good